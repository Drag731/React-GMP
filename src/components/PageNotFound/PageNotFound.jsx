import React from 'react';

import './PageNotFound.scss'

const PageNotFound = () => (
    <div className="b-not-found">
        <div className="b-not-found__wrap">
            <span className="b-not-found__text">404 Page not found</span>
        </div>
    </div>
);

export default PageNotFound;
