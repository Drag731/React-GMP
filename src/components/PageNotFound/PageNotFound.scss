.b-not-found {
  min-height: 300px;
  display: flex;
  justify-content: center;
  align-items: center;
  &__wrap {
    font-size: 30px;
  }
}