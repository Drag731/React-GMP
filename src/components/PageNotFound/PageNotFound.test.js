import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import PageNotFound from './PageNotFound.jsx';

configure({ adapter: new Adapter() });

describe('<PageNotFound  />', () => {
  it('Snapshot one', () => {
    const tree = renderer
      .create(<PageNotFound />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('Renders NoFilmsFound component', () => {
    const tree = shallow(<PageNotFound />);
    expect(tree.find('.b-not-found')).toHaveLength(1);
  });
});
