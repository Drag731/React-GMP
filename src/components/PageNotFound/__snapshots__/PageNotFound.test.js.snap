// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageNotFound  /> Snapshot one 1`] = `
<div
  className="b-not-found"
>
  <div
    className="b-not-found__wrap"
  >
    <span
      className="b-not-found__text"
    >
      404 Page not found
    </span>
  </div>
</div>
`;
